MQuince/MQuince.Services.Tests/AppointmentTests.cs

```diff
@@ -316,7 +318,62 @@ public void Get_free_appointments_return_appointments()
             Assert.True(CompareAppointmentsDTO(output, appointments));
         }
 
-		private bool CompareAppointmentsDTO(IEnumerable<Appointment> output, IEnumerable<AppointmentDTO> appointments)
+        [Fact]
+        public void Cancel_appointment()
+        {
+            AppointmentService _appointmentService = new AppointmentService(CreateStubRepository(), workTimeService);
+
+            Guid appointmentId = Guid.Parse("08d89d75-3533-47ac-80a1-6cd77742dd25");
+
+            _appointmentService.CancelAppointment(appointmentId);
+            IdentifiableDTO<AppointmentDTO> canceledAppointment = _appointmentService.GetById(appointmentId);
+
+            Assert.True(canceledAppointment.EntityDTO.IsCanceled);
+        }
+
+        [Fact]
+        public void Cancel_appointment_when_is_invalid_data_returns_null()
```

Comment:
Test method name is not good, if you return null in this situation you would get null pointer exception. You are returning false, not null.
Also we don't know what data is invalid here.